### ISSUE TITLE
toXlsx does not preserve the bold font in comments

### ISSUE THREAD

**Issue by stla** [qrilka/xlsx#76]
Hello,

 When I comment a cell in Excel, Excel automatically adds my name at the top of the comment, in bold font.

 This is what I get in `comment1.xml`:

 ```
 <commentList>
     <comment ref="A1" authorId="0">
         <text>
             <r>
                 <rPr><b/><sz val="9"/><color indexed="81"/><rFont val="Tahoma"/><charset val="1"/></rPr>
                 <t>Stéphane Laurent:</t>
             </r>
             <r>
                 <rPr><sz val="9"/><color indexed="81"/><rFont val="Tahoma"/><charset val="1"/></rPr>
                 <t xml:space="preserve">
                     hello</t>
             </r>
         </text>
     </comment>
 </commentList>
 ```

When I read the xlsx file with `toXlsx` and when I rewrite it, without modifying anything, my name is not in bold font anymore.

This is the `XlsxText` I get when I read the file:

```
XlsxRichText
  [ RichTextRun
      { _richTextRunProperties =
          Just
            RunProperties
              { _runPropertiesBold = Nothing
              , _runPropertiesCharset = Just 1
              , _runPropertiesColor =
                  Just
                    Color
                      { _colorAutomatic = Nothing
                      , _colorARGB = Nothing
                      , _colorTheme = Nothing
                      , _colorTint = Nothing
                      }
              , _runPropertiesCondense = Nothing
              , _runPropertiesExtend = Nothing
              , _runPropertiesFontFamily = Nothing
              , _runPropertiesItalic = Nothing
              , _runPropertiesOutline = Nothing
              , _runPropertiesFont = Just "Tahoma"
              , _runPropertiesScheme = Nothing
              , _runPropertiesShadow = Nothing
              , _runPropertiesStrikeThrough = Nothing
              , _runPropertiesSize = Just 9.0
              , _runPropertiesUnderline = Nothing
              , _runPropertiesVertAlign = Nothing
              }
      , _richTextRunText = "St\233phane Laurent:"
      }
  , RichTextRun
      { _richTextRunProperties =
          Just
            RunProperties
              { _runPropertiesBold = Nothing
              , _runPropertiesCharset = Just 1
              , _runPropertiesColor =
                  Just
                    Color
                      { _colorAutomatic = Nothing
                      , _colorARGB = Nothing
                      , _colorTheme = Nothing
                      , _colorTint = Nothing
                      }
              , _runPropertiesCondense = Nothing
              , _runPropertiesExtend = Nothing
              , _runPropertiesFontFamily = Nothing
              , _runPropertiesItalic = Nothing
              , _runPropertiesOutline = Nothing
              , _runPropertiesFont = Just "Tahoma"
              , _runPropertiesScheme = Nothing
              , _runPropertiesShadow = Nothing
              , _runPropertiesStrikeThrough = Nothing
              , _runPropertiesSize = Just 9.0
              , _runPropertiesUnderline = Nothing
              , _runPropertiesVertAlign = Nothing
              }
      , _richTextRunText = "\r\nhello"
      }
  ]
```

I think that `_runPropertiesBold` should be `Just True` for the first line.

**Comment by qrilka** [comment]
Hi @stla that seems to be another place that needs to be fixed as a part of #69 
I'll create a fix

**Comment by qrilka** [comment]
Closed by 83907df
